src/In.ProjectEKA.OtpService/Notification/Action.cs

```diff
@@ -2,6 +2,6 @@ namespace In.ProjectEKA.OtpService.Notification
 {
     public enum Action
     {
-        ConsentRequestCreated
+        ConsentRequestCreated, CmidRecovered
```

Comment:
ConsentManagerAccountRecovered in the next line

Reply:
Refactor the code where is required